### DISCUSSION TITLE
Editable font size for article list

### DISCUSSION THREAD

**Discussion by drtn222** [spacecowboy/Feeder#160] [Ideas]
While you can change the font size for the articles themselves, the article list's font is fixed and only depends on your phones text and display size general settings. I think it would be good to have the ability to change the size tin article list too. As someone who needs the reading glasses, if I want to glance at the headlines on the go without reaching for the reading glasses, that would be extremely helpful.

**Comment by spacecowboy** [comment]
Why don't you increase font size globally on your phone?

**Reply by drtn222** [reply]
Did that, but it's inconsistent. Some apps have huge font size and seems like Feeder is on the smaller side. I'm not saying it's impossible to read, by any means. It's still ok. I just think that would be a nice feature to have. It can be somehow connected to the article font size, or independent. I don't know what is easier as I have zero understanding of coding. 

Just an additional perk in any case, it does in no way affect the quality of the app, which is amazing. If possible, I think it would be nice.

**Reply by drtn222** [reply]
Just to elaborate a bit more. If my article view is only titles, it's readable and that's how I keep it. Could do with little bigger, but not a deal breaker. My preferred option would be ten lines, so I can see both the headline and a few lines of text. That I cannot use, as the text of the few first lines would be too small to read comfortably. 

That depends on my needs, which might be very specific though, thus I do realize it might be too niche. I follow a bunch of low quality media, as the media in my country use bad headlines (often attention grabbinng and misleading), but I must keep updated on political news due to my work and those are the news sources, not much I can do about it. 

And lastly, thanks for the reply, I really appreciate your efforts in answering every post.

**Reply by spacecowboy** [reply]
:+1: 

I won't say it won't happen but it's not a priority. I try to use the officially recommended text sizes by default

**Reply by drtn222** [reply]
That's good enough for me :). Hope it will. Thank you!

**Comment by humphreylee** [comment]
I second the proposal. Same situation. Thanks.
